pkg/compliance/rule_event.go

```diff
@@ -0,0 +1,20 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package compliance
+
+// KV defines a key value map for storing attributes of a reported rule event
+type KV map[string]string
```

Comment:
shouldn't we describe the data stored in the map? something like `RuleAttributes` ?

Reply:
we can probably call it RuleProps, but the use case for these is in the reported event rather than input for the rule, so using anything with `Rule` could be misleading

Reply:
I see thanks, I'll let you pick the right name